keel-echo/src/main/kotlin/com/netflix/spinnaker/keel/echo/model/EchoNotification.kt

```diff
@@ -13,8 +13,9 @@ data class EchoNotification(
   val templateGroup: String? = null,
   val severity: Severity,
   val source: Source? = null,
-  val interactiveActions: InteractiveActions?,
-  val additionalContext: Map<String, Any?>? = null
+  val interactiveActions: InteractiveActions? = null,
+  val additionalContext: Map<String, Any?>? = null,
+  val useInteractiveBot: Boolean = true
```

Comment:
@luispollo here is where we default to using the interactive bot for all of keel notifications (not task notifications, just ones sent like this by keel)

Reply:
Cool. We might want to fix the model in echo such that you don't have to specify a whole `interactiveActions` block just to specify the color, but that's icing on the cake. 🙂